Cargo.lock

```diff
@@ -72,7 +72,7 @@ dependencies = [
 [[package]]
 name = "autorust_openapi"
 version = "0.2.0"
-source = "git+https://github.com/ctaggart/autorust_openapi#81f0907f7adcf00cca18c89d09db80362b062096"
+source = "git+https://github.com/ctaggart/autorust_openapi#f21d16e291fd46e956216a04f61d7d3b29ba087a"
```

Comment:
I'll get autorust_openapi updated in another pr. The enum codegen, to be done right, needs https://github.com/ctaggart/autorust_openapi/issues/23.